src/app/themes/theme.js

```diff
@@ -314,6 +314,11 @@ const theme = createTheme({
     MuiTableHead: {
       root: {
         borderRadius: '8px',
+        backgroundColor: '#ffffff',
+        top: '0',
+        position: 'sticky',
```

Comment:
This is a global change to all tables, right? Seems like we should be very careful of making every table in the system have a sticky header, and a new color, and a different zindex, etc.

Reply:
Unless the table has a min height the table header wont be sticky, I thought we should add it in the theme and apply the min height we will feel necessary